calc/src/calc_fee.rs

```diff
@@ -49,6 +49,12 @@ impl Multiplier {
             ("westend", v) if 10 < v && v < 31 => V1((new_i128(inner), false)),
             ("westend", v) if 31 <= v => V2(new_u128(inner)),
 
+            ("statemine", _v) => V2(new_u128(inner)),
+            ("statemint", _v) => V2(new_u128(inner)),
+
+            ("westmine", _v) => V2(new_u128(inner)),
+            ("westmint", _v) => V2(new_u128(inner)),
```

Comment:
Did you consider swapping around the logic a bit here; just check for the weird cases and give every one else `new_i128` by default?

Reply:
I see what you are saying. Kinda like the equivalent of ending the `match` statement with a 

``` rust
/// account for defaults
_ => V1((new_i128(inner), false)),
```

I did think about this, and in all honesty I am kinda on the fence on whether it would be appropriate for `@substrate/calc` to give a default `Multiplier` or not. We enforce third party chains to update their own fee configuration through the controller-configs. Meaning if they don't have a `minCalcFeeRuntime` in the configuration they wont calculate fees anyways. Then meaning they would have to add their match case inside of the rust package as well which allows for zero room for error on our side or assumptions on another chains fee calculation.

But at the same time, as a package it makes sense for the `Multiplier` to have some kind of default value because then it can support more than just Polkadot, Kusama, parity testnets, and parity parachains.